Directory.Build.props

```diff
@@ -10,7 +10,7 @@
         <Product>Microsoft .NET Core</Product>
         <Copyright>$(CopyrightNetFoundation)</Copyright>
         <PackageLicenseExpression>MIT</PackageLicenseExpression>
-        <NoWarn>NU5105;NU5128</NoWarn>
+        <NoWarn>NU5105;NU5128;0419,0649</NoWarn>
```

Comment:
Is this change intentional? Why it is needed?

Reply:
Copied from dotnet/runtime https://github.com/dotnet/runtime/blob/3d8d29731b62c6bdc37c6fb3e22ef8bcdd5ec99b/src/coreclr/System.Private.CoreLib/System.Private.CoreLib.csproj#L46

These rules are conflicting with other rules that we have in `CodeAnalysis.ruleset` and need to be disabled. Looking at how runtime repo handles the situation, I believe these are (rules that have IDs without any letters at the beginning) old analyzer rules and cannot be deactivated using the ruleset file. Even when they are not disabled, their warnings don't show up in VS, they make no code fix suggestions. You can only see them if you run `dotnet build`. They also break the build since we treat warnings as errors.